sdp/processors/base_processor.py

```diff
@@ -101,6 +114,7 @@ def __init__(
         self,
         max_workers: int = -1,
         chunksize: int = 100,
+        in_memory_chunksize: int = 1000000,
```

Comment:
I'd suggest changing some of the tests to have this parameter <100 so that it's actually used during testing in a meaningful way